src/ngx_http_modsecurity_module.c

```diff
@@ -520,6 +539,31 @@ ngx_module_t ngx_http_modsecurity_module = {
 };
 
 
+static ngx_http_variable_t  ngx_http_modsecurity_vars[] = {
+    { ngx_string("modsecurity_req_headers_phase_time"), NULL,
+      ngx_http_modsecurity_req_headers_phase_time, 0,
+      NGX_HTTP_VAR_NOCACHEABLE, 0 },
+
+    { ngx_string("modsecurity_req_body_phase_time"), NULL,
+      ngx_http_modsecurity_req_body_phase_time, 0,
+      NGX_HTTP_VAR_NOCACHEABLE, 0 },
+
+    { ngx_string("modsecurity_resp_headers_phase_time"), NULL,
+      ngx_http_modsecurity_resp_headers_phase_time, 0,
+      NGX_HTTP_VAR_NOCACHEABLE, 0 },
+    
+    { ngx_string("modsecurity_resp_body_phase_time"), NULL,
+      ngx_http_modsecurity_resp_body_phase_time, 0,
+      NGX_HTTP_VAR_NOCACHEABLE, 0 },
+
+    { ngx_string("modsecurity_logging_phase_time"), NULL,
+      ngx_http_modsecurity_logging_phase_time, 0,
+      NGX_HTTP_VAR_NOCACHEABLE, 0 },
+
+      ngx_http_null_variable
```

Comment:
ngx_http_null_variable is introduced in 1.13.4, expand it to support old nginx. https://github.com/nginx/nginx/commit/b992f7259ba4763178f9d394b320bcc5de88818b